project/ProjectSettings.scala

```diff
@@ -27,7 +27,8 @@ object ProjectSettings {
     "ciBuildNumber"    -> sys.env.get("BUILD_NUMBER"),
     "commitSha"        -> commitSha,
     "currentBranch"    -> currentBranch,
-    "interfaceVersion" -> interfaceVersion
+    "interfaceVersion" -> interfaceVersion,
+    "environment"      -> sys.env.get("ENVIRONMENT_TYPE")
```

Comment:
what about injecting the whole base url instead of the environment?
Plus, here "environment" is misleading, because it would assume `api.dev`, `api.uat` or `api`